test/sql/copy/csv/test_csv_httpfs.test

```diff
@@ -0,0 +1,93 @@
+# name: test/sql/copy/csv/test_csv_httpfs.test
+# description: This test triggers the http prefetch mechanism.
+# group: [csv]
+
+require httpfs
+
+query IIIIII rowsort
+SELECT * from read_csv_auto('https://www.data.gouv.fr/fr/datasets/r/6d186965-f41b-41f3-9b23-88241cc6890c');
```

Comment:
These tests will likely break silently at some point either by the files being gone, or the servers supporting range requests.

Ideally we would have a set of test servers with different endpoints one of which does not support range requests. I think that can come in another PR though

Reply:
That is very much true, it can also break from API changes, I'll add a comment on the header of the test,